src/components/landing/sponsors.astro

```diff
@@ -138,7 +155,10 @@ const sponsors = [
           <div class="mb-[64px] relative">
             <div class="flex items-center mb-6">
               <span class="block w-full h-[1px] bg-[#d3d2d6]" />
-              <span class="text-2xl font-semibold px-5">{tier.name}</span>
+              <div class="flex flex-row items-center gap-2 px-5">
+                <Icon name="mdi:crown" class="size-8" style={`color: ${tier.color};`}/>
```

Comment:
I'm not sold on the crowns either, though they are cute

<img width="493" alt="Screenshot 2024-12-03 at 10 30 15 PM" src="https://github.com/user-attachments/assets/4a4edacd-aea6-4846-a764-fc86e3ad9b68">

Reply:
I was trying to find a way to visually distinguish the tiers. The main issue is that visually they are all the same, nothing apart from the words themselves communicate which tier represents a higher donation rank. I thought the colored crown is a subtle way to highlight this difference while also adding some gentle visual flair. Do you have any ideas for other solutions we could explore, or perhaps we could polish this one by changing the colors and the icon used?